system/lib/libc/musl/src/network/recvmsg.c

```diff
@@ -15,7 +15,7 @@ ssize_t recvmsg(int fd, struct msghdr *msg, int flags)
 	}
 #endif
 	r = socketcall_cp(recvmsg, fd, msg, flags, 0, 0, 0);
-#if LONG_MAX > INT_MAX
+#if LONG_MAX > INT_MAX && 0  // XXX EMSCRIPTEN: __pad fields don't exist.
```

Comment:
Are these `&& 0`s temporary workarounds?

Reply:
I don't think they're temporary.

Reply:
How about `&& !defined(__EMSCRIPTEN__)` instead ?   I'm also curious why these fields don't exist if we shared the struct definition with musl?